lib/WebpackConfig.js

```diff
@@ -682,6 +698,20 @@ class WebpackConfig {
         });
     }
 
+    configureLoaderRule(name, callback) {
+        logger.warning('Be careful when using Encore.configureLoaderRule(), this is a low-level method that can potentially breaks Encore and Webpack when not used carefully.');
```

Comment:
Not entirely disagreeing with that warning but it may end up being a bit annoying to see it everytime you build if you have a legitimate reason of using `configureLoaderRule()`.

Reply:
Well, everytime I'm running the dev-server I'm having the warning about the absolute public path and that's fine :man_shrugging: 

Maybe you have a better idea? :confused:

Reply:
Hum, maybe we can display the warning only once, even if we call `configureLoaderRule()` multiple times?

Or just remove it, but keep the warning in the comment?

Reply:
That's not a big issue, we can keep it that way and remove the warning later if it people complain.

Reply:
Okay, nice